cluster/gce/util.sh

```diff
@@ -549,6 +607,8 @@ function kube-up {
     --project "${PROJECT}" \
     --region "${REGION}" -q --format yaml | awk '/^address:/ { print $2 }')
 
+  create-certs $MASTER_RESERVED_IP
```

Comment:
I know it's just mimicking existing, but nit: `${MASTER_RESERVED_IP}` (and below)